Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM python:3.5
-WORKDIR /code
-ADD . /code
```

Comment:
I though it was better to actually run the last tag but I think docker-hub already handles that so we can switch back to master.